src/UKHO.ERPFacade.Common/PermitDecryption/BFAlg.cs

```diff
@@ -0,0 +1,396 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.ERPFacade.Common.PermitDecryption
+{
+    [ExcludeFromCodeCoverage]
+    public class BFAlg : IBlowfishAlgorithm
```

Comment:
Please rename class and file to BlowfishAlgorithm

Reply:
Renamed